### PR TITLE
Recognized Contributors for SIG-Registries

### PR DESCRIPTION
**Pull request by technosophos**
I am nominating the participants of SIG-Registries to become Recognized Contributors. I am nominating this group because I have observed each of these people participate in (proto-)SIG-Registries activities.

https://github.com/bytecodealliance/governance/blob/main/TSC/charter.md#recognized-contributors

Depends on #1

### CODE REVIEW

**Comment by technosophos** [comment]
Can the following people 👍  if you accept this nomination to be a Recognized Contributor, or 👎  to decline the nomination?

Hardock, Brian (@fibonnaci1729)
Hayes, Bailey (@ricochet)
Huene, Peter (@peterhuene)
Kulakowski, George (@kulakowski-wasm)
Matei, Radu (@radu-matei)
McCallum, Nathaniel (@npmmcallum)
Noorali, Michelle (@michelleN)
Thomas, Taylor (@thomastaylor312)
Squillace, Ralph (@squillace)
Sverre, Carl (@carlsverre)
Volosatovd, Roman (@rvolosatovs)
Wagner, Luke (@lukewagner)
Woods, Will (@wgwoods)

If we don't hear from you by March 31, I'll remove you from the PR and we can add you later if you so desire.

Recognized Contributors can vote in certain Bytecode Alliance elections. To be an RC, you must be an active contributor to one of the Bytecode Alliance projects. You can read more here: https://github.com/bytecodealliance/governance/blob/main/TSC/charter.md#recognized-contributors

**Comment by peterhuene** [comment]
:+1:

**Comment by kulakowski-wasm** [comment]
👍

File: recognized-contributors.md [hunk 1]
```diff
@@ -8,16 +8,31 @@ Format of entries: `Surname, First name (GitHub Username)`. When it is tradition
...
+Thomas, Taylor (@thomastaylor312)
+Schneidereit, Till (@tschneidereit)
+Squillace, Ralph (@squillace)
+Sverre, Carl (@carlsverre)
 
 ## U-Z
 
-Wang Xin (xwang98)
\ No newline at end of file
+Volosatovd, Roman (@rvolosatovs)
```
Comment (rvolosatovs) [inline recognized-contributors.md:35]:
```suggestion
Volosatovs, Roman (@rvolosatovs)
```

**Comment by carlsverre** [comment]
👍 looks solid to me! Excited to contribute!

File: recognized-contributors.md (hunk 1 above)
Comment (technosophos) [reply recognized-contributors.md]: Sorry about that.

**Review by xwang98 — APPROVED** [review]

**Comment by squillace** [comment]
👍

**Comment by radu-matei** [comment]
👍

**Comment by fibonacci1729** [comment]
👍
________________________________

**Comment by tschneidereit** [comment]
In today's meeting the TSC approved these nominations, and all added people have accepted the nomination, so this nomination is fully accepted.

**Pull request merged**